test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -393,6 +393,22 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
     assert_equal expected, result.dig(:editions, 0, :translations)
   end
 
+  test "includes unpublishing details" do
+    edition = create(:withdrawn_edition)
+    result = DocumentExportPresenter.new(edition.document).as_json
+
+    expected = {
+      id: edition.unpublishing.id,
+      explanation: edition.unpublishing.explanation,
+      alternative_url: nil,
+      created_at: edition.unpublishing.created_at,
+      updated_at: edition.unpublishing.updated_at,
+      redirect: false,
+    }
+
+    assert_equal expected, result.dig(:editions, 0, :unpublishing)
+  end
```

Comment:
Is it worth writing another test for unpublish-and-redirect just to make sure `redirect` and `alternative_url` are set?